aiida/orm/nodes/data/upf.py

```diff
@@ -25,114 +23,58 @@
 
 __all__ = ('UpfData',)
 
-
 UPFGROUP_TYPE = GroupTypeString.UPFGROUP_TYPE.value
 
-_upfversion_regexp = re.compile(
```

Comment:
Mainly out of curiosity, is there a way to disable yapf in a block of code?
Here the newlines were intended (mode re.VERBOSE) to increase the readability of the regex

Reply:
I actually did this manually. I will restore the formatting and see if yapf keeps it

Reply:
Should be restored now

Reply:
In case you need it, disabling YAPF for a specific piece of code is documented here: https://github.com/google/yapf#why-does-yapf-destroy-my-awesome-formatting